### PR TITLE
Source Google Ads: temporary patch to avoid 500 Internal server error

### PR DESCRIPTION
**Pull request by roman-yermilov-gl**
## What
*The issue: https://github.com/airbytehq/oncall/issues/4306*

## How
*Prepare a temporary patch that will filter out `user_interest.availabilities` and `user_interest.launched_to_all` fields from the **user_interest** stream*

## Recommended reading order
1. `x.python`

### CODE REVIEW

**Review by alafanechere — COMMENTED** [review]
Before merging this:
* Please update the changelog in `google_ads.md`
* Pin our internal connection with source-google-ads to `3.3.2` so that we can monitor when this would be resolved on Google side.

**Review by artem1205 — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py [hunk 1]
```diff
@@ -214,8 +214,12 @@ def get_field_value(field_value: GoogleAdsRow, field: str, schema_type: Mapping[
...
     @staticmethod
-    def parse_single_result(schema: Mapping[str, Any], result: GoogleAdsRow):
+    def parse_single_result(schema: Mapping[str, Any], result: GoogleAdsRow, nullable: Optional[List[str]] = None):
+        if nullable is None:
+            nullable = []
         props = schema.get("properties")
         fields = GoogleAds.get_fields_from_schema(schema)
-        single_record = {field: GoogleAds.get_field_value(result, field, props.get(field)) for field in fields}
+        single_record = {
+            field: GoogleAds.get_field_value(result, field, props.get(field)) if field not in nullable else None for field in fields
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py:223]: I have two questions with this line:
* My understanding of the issue is that we were passing fields that Google Ads would break on. 
* I don't see `get_field_value` using it's third argument `schema_type` so why do we need to change the caller? Shouldn't we just remove the argument from `get_field_value` instead? 

Based on these two questions, why is this change needed?

**Review by maxi297 — COMMENTED** [review]
One concern because I think I don't fully understand the scope of the PR

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py (hunk 1 above)
Comment (maxi297) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py:223]: After a sync discussion, it seems like the values for "user_interest.availabilities" and "user_interest.launched_to_all" have changed following the 500 errors and it is unexpected so until we understand the issue with the 500 errors, we will nullify those two fields so we will add a comment to reflect

**Review by maxi297 — APPROVED** [review]
Given the comment is added, I'm good with the mitigation step!

**Review by bazarnov — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py (hunk 1 above)
Comment (roman-yermilov-gl) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py:229]: Made a comment with explanation what is going on
